apps/indexer/src/api/mappers/proposals.ts

```diff
@@ -19,9 +19,14 @@ export const ProposalsRequestSchema = z.object({
     .optional(),
   orderDirection: z.enum(["asc", "desc"]).default("desc").optional(),
   status: z
-    .string()
+    .union([z.string(), z.array(z.string())])
```

Comment:
This could be the `ProposalStatus`, making the user aware of the possible statuses